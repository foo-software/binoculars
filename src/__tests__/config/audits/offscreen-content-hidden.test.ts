import auditOffscreenContentHidden from '../../../config/audits/offscreen-content-hidden';

describe('offscreen-content-hidden audit', () => {
  it('should match snapshot', () => {
    expect(auditOffscreenContentHidden('en')).toMatchSnapshot();
  });
});
