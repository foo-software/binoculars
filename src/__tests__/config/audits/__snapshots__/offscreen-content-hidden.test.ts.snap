// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`offscreen-content-hidden audit should match snapshot 1`] = `
Object {
  "description": "Offscreen content is hidden with display: none or aria-hidden=true. [Learn more](https://web.dev/offscreen-content-hidden/).",
  "title": "Offscreen content is hidden from assistive technology",
}
`;
